cdap-docs/admin-manual/source/upgrading/cloudera.rst

```diff
@@ -0,0 +1,159 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2016 Cask Data, Inc.
+
+.. _admin-upgrading-cloudera:
+
+===================================
+Upgrading CDAP and Cloudera Manager
+===================================
+
+
+.. _admin-upgrading-cloudera-cdap:
+
+Upgrading CDAP
+==============
+
+Upgrading CDAP Patch Release Versions
+-------------------------------------
+When a new compatible CDAP parcel is released, it will be available via the Parcels page
+in the Cloudera Manager UI.
+
+#. Stop all flows, services, and other programs in all your applications.
+
+#. Stop CDAP services.
+
+#. Use the Cloudera Manager UI to download, distribute, and activate the parcel on all
+   cluster hosts.
+
+#. Start CDAP services.
+
+Upgrading CDAP Major/Minor Release Versions
+-------------------------------------------
+Upgrading between major versions of CDAP involves the additional steps of upgrading the
+CSD, and running the included CDAP Upgrade Tool. Upgrades between multiple Major/Minor
+versions must be done consecutively, and a version cannot be skipped unless otherwise
+noted.
+
+The following is the generic procedure for Major/Minor version upgrades:
+
+#. Stop all flows, services, and other programs in all your applications.
+
+#. Stop CDAP services.
+
+#. Ensure your installed version of the CSD matches the target version of CDAP. For
+   example, CSD version 3.0.* is compatible with CDAP version 3.0.*.  Download `the latest
+   version of the CSD <http://cask.co/resources/#cdap-integrations>`__.
+
+#. Use the Cloudera Manager UI to download, distribute, and activate the corresponding
+   CDAP parcel version on all cluster hosts.
+
+#. Before starting services, run the *Upgrade Tool* to update any necessary CDAP table
```

Comment:
Note: the upgrade tool isn't just for CDAP table definitions. Its broader than that (for instance, schedules, metadata, etc).